charts/internallb-webhook-admission-controller/values.yaml

```diff
@@ -0,0 +1,26 @@
+# Default values for InternalLB Webhook Admission Controller
+rbac:
+  create: true
+
+admissionRegistration:
+  # valid values are "ValidatingWebhookConfiguration", and "MutatingWebhookConfiguration"
+  kind: MutatingWebhookConfiguration
+  path: /mutating-services
+  # valid values are "Ignore" and "Fail"
+  failurePolicy: Ignore
+controller:
+  image: lachlanevenson/internallb-webhook-admission-controller
+  imageTag: 1.9
+  imagePullPolicy: IfNotPresent
+  serviceAccount: internallb-webhook-admission-controller
+  tls:
+    # Admission controller server will inherit this CA from the
+    # extension-apiserver-authentication ConfigMap if available.
+    requestHeaderCA:
+  service:
+    # Type of service; valid values are "LoadBalancer" and "NodePort"
```

Comment:
maybe this is what I was getting wrong? I see that my chart was foolishly missing the service type. Is `ClusterIP` invalid to use here?

Reply:
No that should also be a valid configuration